charts/linux-polska/ezd-crd/1.3.1/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 annotations:
   catalog.cattle.io/certified: partner
   catalog.cattle.io/display-name: CRDs for LP Backend
+  catalog.cattle.io/hidden: "true"
```

Comment:
What is the meaning of this annotation?

Reply:
It causes the chart to not be shown by the UI. In the partner charts repository it is used in only the `linux-polska/ezd-crd` package (this one). I think it is used here because it is effectively a CRD chart for `linux-polska/ezd-backend`.